src/ui/components/Library/Team/View/TestRuns/Overview/RunResults.tsx

```diff
@@ -28,6 +28,10 @@ function TestStatusGroup({ recordings, label }: { recordings: Recording[]; label
     return null;
   }
 
+  const sortedRecordings = recordings.sort((a, b) =>
+    (a.metadata?.test?.title || 0) > (b.metadata?.test?.title || 0) ? 1 : -1
```

Comment:
:hand: Do we want "title", or "filename"?  I'd lean towards files if possible

Reply:
Good call. Let's do filenames! Fixing that :)